### DISCUSSION TITLE
View peaks after pairwise testing

### DISCUSSION THREAD

**Discussion by danieljut** [GreenleafLab/ArchR#748] [Questions / Documentation]
Hi,

After performing pairwise testing between groups is there a way to view which peaks passed the cutoffs in each group? I'm fairly new to this, so perhaps its obvious and I'm just not seeing it. Ideally I want to see a list of peaks that differed between the groups.

Thanks

**Comment by rcorces** [comment]
I think you should familiarize yourself with how a `SummarizedExperiment` works. The peak information is stored within the `SummarizedExperiment` object that is returned by `getMarkerFeatures()` and you can cross reference this with the significance information that is also stored within the `SummarizedExperiment`
